entityDumper/configurationDumper.go

```diff
@@ -0,0 +1,123 @@
+package entityDumper
+
+import (
+	"bufio"
+	"database/sql"
+	"fmt"
+	"github.com/rs/zerolog/log"
+	"github.com/uyuni-project/inter-server-sync/dumper"
+	"github.com/uyuni-project/inter-server-sync/dumper/packageDumper"
+	"github.com/uyuni-project/inter-server-sync/schemareader"
+	"github.com/uyuni-project/inter-server-sync/sqlUtil"
+	"os"
+)
+
+func ConfigTableNames() []string {
+	return []string{
+		"rhnconfigfile",
+		"rhnconfigfilename",
+		"rhnconfigrevision",
+		"rhnconfigcontent",
+		"rhnconfigchannel",
+		"rhnconfigfilestate",
+		"rhnregtokenconfigchannels",
+		"rhnserverconfigchannel",
+		"rhnsnapshotconfigchannel",
+		"susestaterevisionconfigchannel",
+		"rhnconfiginfo",
+		"rhnconfigfilefailure",
+		"rhnchecksum",
+		"rhnchecksumtype",
+		"web_contact",
+	}
+}
+
+func DumpConfigs(options ChannelDumperOptions) {
+	var outputFolderAbs = options.GetOutputFolderAbsPath()
+	db := schemareader.GetDBconnection(options.ServerConfig)
+	defer db.Close()
+	//file, err := os.OpenFile(outputFolderAbs + "/sql_statements.sql", os.O_APPEND | os.O_WRONLY | os.O_CREATE, 0600)
+	file, err := os.Create(outputFolderAbs + "/configurations.sql")
+	if err != nil {
+		log.Fatal().Err(err).Msg("error creating sql file")
+		panic(err)
+	}
+	defer file.Close()
+	bufferWriter := bufio.NewWriter(file)
+	defer bufferWriter.Flush()
+
+	bufferWriter.WriteString("BEGIN;\n")
```

Comment:
This will create two different transations, and we want to package all in one transation.
The file creation and also the begin and commit maybe needs to be placed on the super export command.